daemon/openrazer_daemon/hardware/accessory.py

```diff
@@ -196,3 +196,19 @@ class RazerChargingPadChroma(_RazerDeviceBrightnessSuspend):
                'set_fully_charged_wave', 'set_fully_charged_static', 'set_fully_charged_spectrum', 'set_fully_charged_none', 'set_fully_charged_breath_random', 'set_fully_charged_breath_single', 'set_fully_charged_breath_dual']
 
     DEVICE_IMAGE = "https://dl.razerzone.com/Images/ChargingPadChroma/CPC-ProductImage.png"
+
+
+class RazerLaptopStandV1Chroma(_RazerDeviceBrightnessSuspend):
+    """
+    Class for the Razer Laptop Stand v1 Chroma
+    """
+
+    USB_VID = 0x1532
+    USB_PID = 0x0F0D
+    HAS_MATRIX = True
+    MATRIX_DIMS = [1, 16]
+    METHODS = ['get_device_type_accessory', 'set_static_effect', 'set_wave_effect', 'set_spectrum_effect',
+               'set_none_effect', 'set_breath_random_effect', 'set_breath_single_effect', 'set_breath_dual_effect',
+               'set_custom_effect', 'set_key_row']
+
+    DEVICE_IMAGE = "https://assets3.razerzone.com/Pk-t5-8EwFtS1D2Ct7ZcHoby_Gg=/1500x1000/https%3A%2F%2Fhybrismediaprod.blob.core.windows.net%2Fsys-master-phoenix-images-container%2Fhfe%2Fhbc%2F9081459376158%2FRazer-Laptop-Stand-Chroma-04.jpg"
```

Comment:
Use https://hybrismediaprod.blob.core.windows.net/sys-master-phoenix-images-container/hfe/hbc/9081459376158/Razer-Laptop-Stand-Chroma-04.jpg instead of this proxy